test/e2e/tests/send-edit.spec.js

```diff
@@ -54,12 +54,12 @@ describe('Editing Confirm Transaction', function () {
 
         // has correct updated value on the confirm screen the transaction
         await driver.waitForSelector({
-          css: '.transaction-detail-item:nth-of-type(1) h6:nth-of-type(2)',
-          text: '0.0008 ETH',
+          css: '.currency-display-component__text',
+          text: '0.0008',
         });
         await driver.waitForSelector({
-          css: '.transaction-detail-item:nth-of-type(2) h6:nth-of-type(2)',
-          text: '2.2008 ETH',
```

Comment:
it seems that before we were also checking the total amount.. Is there a reason to have dropped this check?

Reply:
Had a chat with @bschorchit and right now we want the fee details just shows the fees, not the fees + amount.